flexmeasures/data/models/reporting/__init__.py

```diff
@@ -32,7 +32,7 @@ def _compute(self, **kwargs) -> List[Dict[str, Any]]:
 
         for result in results:
             # checking that the event_resolution of the output BeliefDataFrame is equal to the one of the output sensor
-            assert (
+            assert check_output_resolution or (
```

Comment:
This results in skipping the check when `check_output_resolution is True` and doing the check when it is `False`. I expected the opposite.

Please amend the docstring to explain.